SDL/Triplet.h

```diff
@@ -466,6 +487,44 @@ namespace SDL {
     //Cut #3: rt-z pointed
 
     pass = pass and (kZ >= 0) && (rtOut >= rtLo) && (rtOut <= rtHi);
+
+    float rt_InLo = mdsInGPU.anchorRt[firstMDIndex];
+    float rt_InOut = mdsInGPU.anchorRt[secondMDIndex];
+
+    float sdIn_alpha = __H2F(segmentsInGPU.dPhiChanges[innerSegmentIndex]);
+
+    float tl_axis_x = mdsInGPU.anchorX[thirdMDIndex] - mdsInGPU.anchorX[firstMDIndex];
+    float tl_axis_y = mdsInGPU.anchorY[thirdMDIndex] - mdsInGPU.anchorY[firstMDIndex];
+
+    betaIn = sdIn_alpha - SDL::phi_mpi_pi(acc, SDL::phi(acc, tl_axis_x, tl_axis_y) - mdsInGPU.anchorPhi[firstMDIndex]);
+
+    float betaInRHmin = betaIn;
+    float betaInRHmax = betaIn;
+
+    float swapTemp;
+
+    if (alpaka::math::abs(acc, betaInRHmin) > alpaka::math::abs(acc, betaInRHmax)) {
+      swapTemp = betaInRHmin;
+      betaInRHmin = betaInRHmax;
+      betaInRHmax = swapTemp;
+    }
+
+    float sdIn_dr = alpaka::math::sqrt(acc,
+                                       (mdsInGPU.anchorX[secondMDIndex] - mdsInGPU.anchorX[firstMDIndex]) *
+                                               (mdsInGPU.anchorX[secondMDIndex] - mdsInGPU.anchorX[firstMDIndex]) +
+                                           (mdsInGPU.anchorY[secondMDIndex] - mdsInGPU.anchorY[firstMDIndex]) *
+                                               (mdsInGPU.anchorY[secondMDIndex] - mdsInGPU.anchorY[firstMDIndex]));
+    float sdIn_d = rt_InOut - rt_InLo;
+
+    float dr = alpaka::math::sqrt(acc, tl_axis_x * tl_axis_x + tl_axis_y * tl_axis_y);
+    const float corrF = 1.f;
+    betaInCut =
+        alpaka::math::asin(
+            acc, alpaka::math::min(acc, (-sdIn_dr * corrF + dr) * SDL::k2Rinv1GeVf / SDL::ptCut, SDL::sinAlphaMax)) +
+        (0.02f / sdIn_d);
+
+    //Cut #6: first beta cut
```

Comment:
```suggestion
    //Cut #4: first beta cut
```
or is there a reason to skip the cut index?